src/identity/SolidOidcIdentityProvider.ts

```diff
@@ -0,0 +1,87 @@
+import { Provider } from 'oidc-provider';
+
+/**
+ * An Extension of node-oidc-provider with configurations specifically for
+ * Solid OIDC
+ */
+export default class SolidOidcIdentityProvider extends Provider {
+  public constructor() {
+    // TODO (@jackson) [>1.0.0]: finish this.
+    // Also decide which of these should be configurable
+    super('ISSUER_URL', {
+      clients: [
+        // {
+        //   client_id: 'oidcCLIENT',
+        //   client_secret: '...',
+        //   grant_types: ['refresh_token', 'authorization_code'],
+        //   redirect_uris: ['http://sso-client.dev/providers/7/open_id', 'http://sso-client.dev/providers/8/open_id'],
+        // }
+      ],
+      interactions: {
+        policy: [],
+        url(ctx): string {
+          // eslint-disable-line no-unused-vars
```

Comment:
Which var?

Reply:
Guess that's leftover from something else. This is messy code. I will be cleaning it up.